Source/Extensions.swift

```diff
@@ -129,6 +217,44 @@ internal extension UIColor {
         self.init(red:red, green:green, blue:blue, alpha:alpha)
     }
     
+    ///Custom colors
+    
```

Comment:
estos colores custom es mejor que sean propiedades y no funciones =>

```suggestion
    static let lightBlue = UIColor(red: 0.29, green: 0.56, blue: 0.89, alpha: 0.8)
```

Reply:
Listo!